Add test and example for published Agent. (#54477)
Azure/azure-sdk-for-net: sdk/ai/Azure.AI.Projects.OpenAI/README.md

OpenAIResponse response = await responseClient.CreateResponseAsync(items, responseCreationOptions);
```

### Published Agents

Published Agents are available outside the Microsoft Foundry and can be used by external applications.

#### Publish Agent

1. Click **New foundry** switch at the top of Microsoft Foundry UI.
2. Click **Build** at the upper right.
3. Click **Create agent** button and name your Agent.
4. Select the created Agent at the central panel and click **Publish** at the upper right corner.

After the Agent is published, you will be provided with two URLs
- `https://<Fondry Name>.services.ai.azure.com/api/projects/<Project Name>/applications/<Agent Name>/protocols/activityprotocol?api-version=2025-11-15-preview`
- `https://<Fondry Name>.services.ai.azure.com/api/projects/<Project Name>/applications/<Agent Name>/protocols/openai/responses?=2025-11-15-preview`

> The second URL can be usedto call responses API, we will use it to run sample.

Reviewer: ```suggestion
The second URL is the endpoint for the Responses API, and we’ll use it to run the sample.
```

Author: I have rewritten this part.
"These URLs can be used to call responses API after removing path following `/protocols`, resulting in `https://<Fondry Name>.services.ai.azure.com/api/projects/<Project Name>/applications/<Agent Name>/protocols`."

### Use the published Agent
The URL, returned during Agent publishing contains `/openai/responses` path and query parameter, setting `api-version`. These parts need to be removed by `CleanEndpoint` method.

```C# Snippet:Sample_CleanUri_PublishedAgent
private static Uri CleanEndpoint(string endpoint)
{
    Uri uriEndpoint = new(endpoint);
    // Remove the Query part.

>     if (!string.IsNullOrEmpty(uriEndpoint.Query))

Reviewer: In my opinion, it is weird to suggest a code snip to clean up the endpoint.   As long as you say which part to be extract, we should be good.   I just express my feelings.   Not a blocker.

Author: Removed.
